mig-api/api.go

```diff
@@ -70,6 +71,13 @@ func main() {
 	r := mux.NewRouter()
 	s := r.PathPrefix(ctx.Server.BaseRoute).Subrouter()
 
+	postHeartbeat := agents.NewUploadHeartbeat(
+		agents.NewPersistHeartbeatPostgres(&ctx.DB),
+		agents.NewNilAuthenticator())
+
+	// Endpoints that replace previously direct-to-rabbitmq communications.
+	s.Handle("/heartbeat", postHeartbeat).Methods("POST")
```

Comment:
Originally these heartbeat messages required auth to rabbitmq, so should we maintain that requirement (that an agent needs to auth somehow to post a heartbeat?)

Reply:
I don't think it's really worth finding a way to authenticate agents in this case. If an agent or some actor imitating an agent pushes a heartbeat, they aren't going to be able to do anything really damaging. I think finding a way to authenticate agents for this purpose and implementing it would be far more work than it's worth.